### ISSUE TITLE
Monitor test duration

### ISSUE THREAD

**Issue by tfardet** [InseeFrLab/pynsee#185]
I'd propose to add ``--durations=0`` to ``pytest`` so we know how long the tests take to run and we can more easily figure out where changes in test durations come from (cf. recent events).

PS: @hadrilec I noticed that tests are again checking all python versions (d5be92bb025ea0bf735b7c399ca9b52fd5d05fc6), is that on purpose?

**Comment by hadrilec** [comment]
I agree with the proposal, I splitted the tests accross several python versions again as I tried to get the total test duration down

**Comment by tfardet** [comment]
OK, good, you can just include it in any of the open PR, then (changing [``pkgTests_pull_requests.yml#L56``](https://github.com/InseeFrLab/pynsee/blob/master/.github/workflows/pkgTests_pull_requests.yml#L56) and [``pkgTests.yml#L54``](https://github.com/InseeFrLab/pynsee/blob/master/.github/workflows/pkgTests.yml#L54))

**Comment by tfardet** [comment]
given that the tests now take a lot less time I'm not sure this is necessary anymore... any opinions?

**Comment by tgrandje** [comment]
I agree with you
